plural/helm/console/templates/proxies.yaml

```diff
@@ -24,7 +24,7 @@ metadata:
 {{ include "console.labels" . | indent 4 }}
 spec:
   type: sh
-  target: statefulset/console
```

Comment:
This should actually be `deployment/console`. The console used to be a statefulset but it was changed to a deployment. The `plural-console` statefulset is the postgres database.

Reply:
done